directives/questionnaire.py

```diff
@@ -74,7 +74,9 @@ def run(self):
 
         env.questionnaire_is_feedback = is_feedback
         env.question_count = 0
+        env.single_question_points = None
```

Comment:
The `env` variables should be prefixed with the extension name `aplus_` since `env` is shared between all extensions. We have to minimize the risk that other extensions use the same names. So, `env.single_question_points` -> `env.aplus_single_question_points`.